src/test/java/org/springframework/data/redis/serializer/GenericJackson2JsonRedisSerializerUnitTests.java

```diff
@@ -438,14 +441,34 @@ public void internalObjectMapperCustomization() {
 	}
 
 	@Test // GH-2601
-	public void configureWithNullConsumerThrowsIllegalArgumentException() {
```

Comment:
I just removed according to sonarlint plugin. It looks like an unnecessary keyword.